macros/contexts/contextFraction.pl

```diff
@@ -205,6 +205,20 @@ =head1 DESCRIPTION
 default in all four contexts.  You should not set both
 C<requirePureFractions> and C<requireProperFractions> to 1.
 
+=item S<C<< fractionTolerance >>>
+
+Thie determines the tolerance to use when comparing a fraction to a
+real number.  The fraction will be converted to a real, and then this
+is used as the tolerance in a relative-tolerance comparison of the two
+reals.  The default is 10E-10, meaning the decimal must match to
+roughly 10 digits.
```

Comment:
I think you mean `1E-10` here.  In the code you use `10**-10` which is equal to `1E-10`.  Furthermore, that would actually mean roughly 10 digits of accuracy, while `10E-10` only means roughly 9 digits of accuracy.

Not that you don't know that.  Probably just a typo, or mental conversion error from the `**` notation to the `E` notation.

Reply:
I made a lot minute change from `10**-10`, which just didn't look good with the two 10s and negative sign, and didn't change it correctly.  I will fix.

Reply:
Another minor typo, not from this PR is on line 50 of contextFraction.pl. `LimiteFraction`.  Since you have this open, instead of making another PR.